ActindoWorkFlow/UserInterface/01_WorkflowElements.md

```diff
@@ -0,0 +1,203 @@
+
+![Settings side bar](/Assets/Screenshots/ActindoWorkFlow/Workflows/SettingsSideBar02.png "[Settings side bar]")
+
+The settings side bar is located to the right of the workspace and displays the following information:
+
+  - (Action label and name)  
+  The action label can be changed in the *Label* entry field. The action name provides a detailed description of the task to be performed and is read-only.
+
+  - *Key*  
+   Element key. This information is read-only.
+
+  - *Label*  
+  Action label. Click this field to edit the action label. The action label is changed both in the settings side bar and in the diagram.
+
+  - *Queue type*  
+  Click the drop-down list to select the applicable queue type. For detailed information about queue types, see [Configure the queue types](#to-be-completed).
+
+  - *Task event*  
+  Click the drop-down list to select the applicable task event.
+
+  > [Info] This option is linked to the *Tasks* module. It is only displayed if the *Tasks* module is installed and the user has the necessary rights. The *ActindoWorkflow Process Execution Failed* task is preconfigured per default. A list of all existing task events is displayed in the *Events* menu entry of the *Tasks* module.
+
+  - *Configuration*  
+  Depending on the action, there might be configuration options, for instance, defining an email where a receipt or document must be sent. Editing the action configuration requires advanced programming knowledge.
+
+  > [Info] Not all actions have configuration options. In this case, the *Configuration* field is not displayed.
+
+  - *Static inputs*  
+  The optional inputs are displayed. They can be edited by clicking the ![Add](/Assets/Icons/Plus01.png "[Add]") (Add) button.
+
+[comment]: <> (The static inputs determine the data type that can be input. They are preconfigured but can be further configured by the user. Only JSON values are accepted.)
+
+
+## Input and output ports
+
+![Input and output ports](/Assets/Screenshots/ActindoWorkFlow/Workflows/InputOutputPorts.png "[Input and output ports]")
+
+Every action can contain any number of input and output ports. They can be defined with a descriptive name, such as reporter, title or event, or be simply marked as *in* (input) and *out* (output) followed by a consecutive number.
+
+[comment]: <> (The plan is to provide all input and output ports with a name)
+
+Input ports are positioned to the left of the action and output ports are positioned to the right of the action. Both are represented by a square.
+
+Both input and output ports can be of two types:
+
+- **Mandatory**  
+  Mandatory ports must be linked to a place, which in turn must be linked to an action. Otherwise, the workflow stops and shows an error. Places linked to a mandatory port cannot be deleted or removed. When adding a new action, mandatory input/output places are already displayed.
+
```

Comment:
It should actually be "process", right?: Otherwise, the **workflow** stops and shows an error.